### ISSUE TITLE
Can't play music in the voice channel

### ISSUE THREAD

**Issue by Abirul** [discordjs/discord.js#3282]
`const ytdl = require("ytdl-core");

exports.run = async(client, message, args, ops) =>{

    if(!message.member.voiceChannel) return message.channel.send("Please connect a voice channel.");

    if(message.guild.me.voiceChannel) return message.channel.send("Sorry, the bot is already connected to the guild.");

    if(!args[0]) return message.channel.send("Sorry, please input a url following the command.");

    let validate = await ytdl.validateURL(args[0]);

    if(!validate) return message.channel.send("Sorry, please input a **valid** url following the command");

    let info = await ytdl.getInfo(args[0]);

    let voiceConnection = message.member.voiceChannel.join()
    .then(voiceConnection => {
    const streamOptions = { seek: 0, volume: 1, quality: "highest" };
    const stream = ytdl(args[0], { filter : 'audioonly' });
    const streamDispatcher = voiceConnection.playStream(stream, streamOptions);
    })
    .catch(console.error);

    message.channel.send(`Now playing: ${info.title}`);
}`
TypeError [ERR_INVALID_ARG_TYPE]: The "file" argument must be of type string. Received type object
    at validateString (internal/validators.js:107:11)
    at normalizeSpawnArguments (child_process.js:407:3)
    at Object.spawn (child_process.js:543:16)
    at new FfmpegProcess (F:\Abirul\Bot\Music Bot - Copy\node_modules\prism-media\src\transcoders\ffmpeg\FfmpegProcess.js:14:33)
    at FfmpegTranscoder.transcode (F:\Abirul\Bot\Music Bot - Copy\node_modules\prism-media\src\transcoders\ffmpeg\Ffmpeg.js:34:18)
    at MediaTranscoder.transcode (F:\Abirul\Bot\Music Bot - Copy\node_modules\prism-media\src\transcoders\MediaTranscoder.js:27:31)
    at Prism.transcode (F:\Abirul\Bot\Music Bot - Copy\node_modules\prism-media\src\Prism.js:13:28)
    at AudioPlayer.playUnknownStream (F:\Abirul\Bot\Music Bot - Copy\node_modules\discord.js\src\client\voice\player\AudioPlayer.js:97:35)
    at VoiceConnection.playStream (F:\Abirul\Bot\Music Bot - Copy\node_modules\discord.js\src\client\voice\VoiceConnection.js:478:24)
    at F:\Abirul\Bot\Music Bot - Copy\commands\play.js:27:46

**Comment by amishshah** [comment]
Can you tell us which version of `discord.js`, `prism-media` and `ytdl-core` you have installed?

**Comment by amishshah** [comment]
Closing due to inactivity

**Comment by blackVanilla** [comment]
Found solution here: https://github.com/discordjs/discord.js/issues/3375
tl;dr: remove `ffmpeg-binaries`
